package dmitriy.losev.core.exception

const val SAFE_CALL_FAIL = -1
const val BAD_REQUEST = 400
const val INTERNAL_SERVER_ERROR = 500
const val NO_INTERNET = 666
const val RESULT_MAP_EXCEPTION_CODE = 0